code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -490,6 +490,98 @@
 	..()
 	new /obj/item/stack/material/phoron(get_turf(holder.my_atom), created_volume)
 
+/datum/chemical_reaction/uraniumsolidification
+	name = "Solid Uranium"
+	result = null
+	required_reagents = list(/datum/reagent/iron = 5, /datum/reagent/uranium = 20)
+	result_amount = 1
+	minimum_temperature = (-80 CELCIUS) - 100
```

Comment:
Wait - does this mean that at -190 Celsius, it would be _too cold for it to freeze_?

Reply:
Yep. I just used the same values bay threw in for phoron solidification, so blame them for that brilliant piece of logic. Or me for being lazy, either works. Can set them all to 0 if need be.

Reply:
Just having it blank to default to 0 would be preferable, yeah.